extensions/binding/org.eclipse.smarthome.binding.dmx/src/main/java/org/eclipse/smarthome/binding/dmx/internal/action/FadeAction.java

```diff
@@ -0,0 +1,152 @@
+/**
+ * Copyright (c) 2014-2017 by the respective copyright holders.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.binding.dmx.internal.action;
+
+import org.eclipse.smarthome.binding.dmx.internal.Util;
+import org.eclipse.smarthome.binding.dmx.internal.multiverse.Channel;
+import org.eclipse.smarthome.core.library.types.PercentType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link FadeAction} fades a given channel from its current state to the requested
+ * state in the given amount of time. After the fade, the new state is held for
+ * a given or indefinite time.
+ *
+ * @author Davy Vanherbergen
```

Comment:
If we want to integrate the code into ESH, not only you (Jan N. Klug) who submitted the PR has to sign the Eclipse contributions agreement, but also Davy Vanherbergen. Do you know how to contact him?